examples/jfrog-platform/values/values-artifactory-xray.yaml

```diff
@@ -0,0 +1,53 @@
+# An example of installing Artifactory and Xray only (disabling the rest)
+
+### Install
+# helm repo add jfrog https://charts.jfrog.io
+# helm repo update
+# helm upgrade --install jfp jfrog/jfrog-platform -f ./values-artifactory-xray.yaml
+
+### Uninstall
+# helm uninstall jfp
+# kubectl get pvc ...
+# kubectl delete pvc ...
+
+global:
+  joinKey: 00EEEEEEEEEEEEEEEEEEEEEEEEEEE123
```

Comment:
@eldada  will add a comment , to generate `openssl rand -hex 32`

Reply:
Done

Reply:
@eldada  we recommend hex 32 instead of hex 16